### PR TITLE
React capstone project

### PR DESCRIPTION
**Pull request by Gleennkar**
## Web app which can analysis or give details on various companies and their stocks
- setup my project by creating the react app and installing the various Library needed
- Got data from the Financial Modeling api
- Check for linter errors
- Updated README.md
- Passing test

Link to my Loom video : https://www.loom.com/share/d5be0453dec4415ebef2da2f5648d665

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+code {
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    'Courier New',
+    monospace;
+}
```
Comment (Davidosky007) [inline src/index.css]: * Kindly follow the design guidelines which include font families( Lato, Gill sans) as its [required](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=You%20should%20follow,space%20between%20elements.)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Gleennkar 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* Nice looking readme file 👍 
* No linter errors 👍 
* Profesional video presentation 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    Lato,
+    'Gill Sans',
+    'Gill Sans MT',
+    Calibri,
+    'Trebuchet MS',
+    sans-serif;
```
Comment (brytebee) [inline src/index.css:11]: - As pointed out by the previous reviewer, the font do not yet conform to the [design guideline.](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) 
- When I inspect your page I get this font family returned:

Your Design | Template
:---: | :---:
image | image.

- Note that you could target the classes holding your navigation (control) with the Lato font family as well as the card holding the body content with the Gill sans font family.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Gleennkar👋, 

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️

##

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @Gleennkar,

Good job on making the changes asked by the previous reviewers,

Your project is complete!  

## Highlights:
✅ Used Gitflow correctly 👍 
✅ Linter checks passed 👍 
✅ App is hosted properly and works fine as expected 👍 

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**